src/routes/product-list/index.tsx

```diff
@@ -25,22 +26,32 @@ export const ProductAnalyticsPage = (props: Props) => {
     queryFn: () => getProductList(site),
   })
 
+  const categoryQuery = useQuery({
+    queryKey: ["categories", site],
+    queryFn: () => getCategoryList(site),
+  })
+
   let products = query.data ?? []
 
   if (categoryId) {
     products = products.filter((product) => product.category.id === categoryId)
   }
 
+  const currentCategoryName =
+    categoryId && categoryQuery.data?.find((c) => c.id === categoryId)?.name
```

Comment:
Displays product categories we are currently in, rather than always showing "All products"